templates/Includes/Carousel.ss

```diff
@@ -24,10 +24,10 @@
 
                   <%-- Play or Pause --%>
                   <div class="carousel-play-controls">
-                    <button type="button" class="btn btn-link btn-sm" id="carousel-pause" aria-label="Autoplaying carousel with slides. <%t CwpCarousel.PAUSESLIDE "Click to pause carousel slides." %>">
+                    <button type="button" class="btn btn-sm btn-controls" id="carousel-pause" aria-label="Autoplaying carousel with slides. <%t CwpCarousel.PAUSESLIDE "Click to pause carousel slides." %>">
```

Comment:
Maybe `carousel-control` could replace `btn-controls` here, as the styles are specific to the carousel.